extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace Wikia\ContentReview;
+
+use Wikia\ContentReview\Helper;
+use Wikia\ContentReview\Models\CurrentRevisionModel;
+use Wikia\ContentReview\Models\ReviewModel;
+
+class Hooks {
+	const CONTENT_REVIEW_MONOBOOK_DROPDOWN_ACTION = 'content-review';
+
+	public static function onGetRailModuleList( Array &$railModuleList ) {
+		global $wgCityId, $wgTitle;
+
+		if ( self::userCanEditJsPage() ) {
+			$pageStatus = \F::app()->sendRequest(
+				'ContentReviewApiController',
+				'getPageStatus',
+				[
+					'wikiId' => $wgCityId,
+					'pageId' => $wgTitle->getArticleID(),
+				],
+				true
+			)->getData();
+
+			$railModuleList[1403] = [ 'ContentReviewModule', 'Render', [
+				'pageStatus' => $pageStatus,
+				'latestRevisionId' => $wgTitle->getLatestRevID(),
+			] ];
+		}
+
+		return true;
+	}
+
+	public static function onMakeGlobalVariablesScript( &$vars ) {
+		$helper = new Helper();
+
+		$vars['contentReviewExtEnabled'] = true;
+		$vars['contentReviewTestModeEnabled'] = $helper->isContentReviewTestModeEnabled();
+		$vars['contentReviewScriptsHash'] = $helper->getSiteJsScriptsHash();
```

Comment:
Let's be consistent with JS global variables naming convention and use `wg` prefix